dbid.env

```diff
@@ -0,0 +1 @@
+EVENTUATE_OUTBOX_ID=1
```

Comment:
Please rename this file to `docker-compose-env-files/db-id-gen.env`. 
Ditto for `empty.env`

Reply:
ok